src/main/java/com/github/dregheap/DregHeapBackendApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.github.dregheap;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DregHeapBackendApplication {
```

Comment:
Don't use words like 'backend', 'class' and so on in class namings. And don't use word 'application'. Just 'DregHeap' will be enough.